src/templates/header.html

```diff
@@ -25,18 +25,17 @@
 
 <mkt-header id="site-header" class="header site-header">
   {# TODO: remove IDs and classes when we stop using <header>. #}
-  <a href="#" class="header-button icon back mkt-header--back"
-     title="{{ _('Back') }}">
-    <b>{{ _('Back') }}</b>
-  </a>
   <mkt-nav-toggle for="nav">
-    <button class="hamburger" title="{{ _('Toggle Menu') }}"></button>
+    <button class="hamburger" title="{{ _('Toggle Menu') }}">
```

Comment:
We lose the title for the back arrow since this is now the hambacker instead of separate buttons. Would it be better to update the title and some class names here on `navigate`?